server/controllers/inventory/inventory/core.js

```diff
@@ -78,14 +78,28 @@ function updateItemsMetadata(record, identifier) {
   if (record.group_uuid) {
     record.group_uuid = db.bid(record.group_uuid);
   }
+ 
+
+  let sql = 'UPDATE inventory SET ? WHERE uuid = ?;';
+  // if there is no property to update this query won't work
+
+  if(util.isEmpty(record)) { //there is no change, but user has submitted
+    sql = 'UPDATE inventory SET uuid = ? WHERE uuid = ?;';
```

Comment:
Instead of executing this query, why not just return `getItemsMetadataById()`?

Also, we should probably block a user from sending an empty request on the client.